snapcraft/how-to/quickstart/how-snapcraft-builds-a-snap.md

```diff
@@ -0,0 +1,90 @@
+# How Snapcraft builds snaps
+The Snapcraft tool can perform a variety of operations, including:
+
+* Snap creation operations like build, stage, prime, pack, etc.
+* Store management operations like login, register, sign-build, etc.
+* Extensions-specific operations like expand-extensions and list-extensions.
+
+These operations translate into practical tasks like:
+
+* It can build snaps locally or send remote tasks to Launchpad.
+* It allows the developer to register and log into the Snap Store.
+* It can upload snaps to the Snap Store.
+* It can promote snaps to different channels.
```

Comment:
I would remove the "It" from these lines; it's well established we're talking about Snapcraft, and it better mirrors the previous lines (4-6).